server/runtime_status.go

```diff
@@ -15,31 +19,26 @@ func (s *Server) Status(ctx context.Context, req *pb.StatusRequest) (resp *pb.St
 		recordError(operation, err)
 	}()
 
-	// Deal with Runtime conditions
-	runtimeReady, err := s.Runtime().RuntimeReady()
-	if err != nil {
-		return nil, err
+	runtimeCondition := &pb.RuntimeCondition{
+		Type:   pb.RuntimeReady,
+		Status: true,
 	}
-	networkReady, err := s.Runtime().NetworkReady()
-	if err != nil {
-		return nil, err
+	networkCondition := &pb.RuntimeCondition{
+		Type:   pb.NetworkReady,
+		Status: true,
 	}
 
-	// Use vendored strings
-	runtimeReadyConditionString := pb.RuntimeReady
-	networkReadyConditionString := pb.NetworkReady
+	if err := s.netPlugin.Status(); err != nil {
```

Comment:
This `Status` call is a strange interface, providing no way to distinguish between “I got the status, and it's ‘not ready yet’” and “I couldn't figure out the status, it may be ready but I don't know”.  It looks like the only error it could return is from [here][1] although the error from [here][2] is likely to be more useful.  But that's an ocicni limitation, so I'm fine with this as it stands for this PR.

[1]: https://github.com/cri-o/ocicni/blob/9b451e26eb7c694d564991fbf44f77d0afb9b03c/pkg/ocicni/ocicni.go#L302-L303
[2]: https://github.com/cri-o/ocicni/blob/9b451e26eb7c694d564991fbf44f77d0afb9b03c/pkg/ocicni/ocicni.go#L279-L283